e2e/nomostest/testpredicates/predicates.go

```diff
@@ -1353,6 +1386,49 @@ func ClusterRoleBindingSubjectNamesEqual(subjects ...string) func(o client.Objec
 	}
 }
 
+// And returns a Predicate that succeeds only if all of the Predicates succeed.
+// If any one of the specified Predicates errors, the And Predicate errors.
+// Logs the number of predicates, if successful.
+func And(logger *testlogger.TestLogger, predicates ...Predicate) Predicate {
```

Comment:
Is `And` needed? This is basically the default behavior if you pass in a list of predicates right?

Reply:
Removed